apps/etterlatte-behandling/src/test/kotlin/behandling/klage/KlageServiceImplTest.kt

```diff
@@ -254,6 +285,87 @@ internal class KlageServiceImplTest : BehandlingIntegrationTest() {
         }
     }
 
```

Comment:
Ønsker meg en test på at sletting av oversendelse / vedtaksbrev ikke kræsjer ferdigstilling! :D

Reply:
I'm on it